style.css

```diff
@@ -0,0 +1,201 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+}
+
+body {
+  background: #e5e5e5;
+}
+
```

Comment:
- [ ] Your design looks good but not better, I think you can still improve it, please take a look at these 2 images right here and implement the following mentioned changes.
- [ ] Take a look closer at the text in the red border, we can see that the font and the alignment are different, please update your design using the same and correct Figma properties
- [ ] At the bottom of the page, there is a left-border-radius which is not in your design, please fix that one to as we move onto the next phases of the project
> |Figma on the left and your design on the right |
> |---|
> | <img width="728" alt="two" src="https://user-images.githubusercontent.com/77843050/204899532-0f5fa633-879b-43c3-8912-a5d5743ac8ce.png"> |